### PR TITLE
feat: Add Fortran icons

### PR DESCRIPTION
**Pull request by Orest58008**

### CODE REVIEW

**Comment by MartinFillon** [comment]
Hey please format your code

**Comment by Orest58008** [comment]
Oh, I need to unformat it back

File: src/output/icons.rs [hunk 1]
```diff
@@ -8,111 +8,112 @@ struct Icons;
...
+    const TEXT: char =            '\u{f15c}'; // 
+    const TYPST: char =           '\u{1D42D}'; // 𝐭
+    const UNITY: char =           '\u{e721}'; // 
+    const VECTOR: char =          '\u{f0559}'; // 󰕙
+    const VIDEO: char =           '\u{f03d}'; // 
+    const VIM: char =             '\u{e7c5}'; // 
+    const WRENCH: char =          '\u{f0ad}'; // 
+    const XML: char =             '\u{f05c0}'; // 󰗀
+    const YAML: char =            '\u{e6a8}'; // 
+    const YARN: char =            '\u{e6a7}'; // 
```
Comment (MartinFillon) [inline src/output/icons.rs:116]: please revert this

**Review by MartinFillon — CHANGES_REQUESTED** [review]

File: src/output/icons.rs (hunk 1 above)
Comment (Orest58008) [reply src/output/icons.rs:116]: Oh, I unformatted it incorrectly. Sorry for being so incompetent, I have little experience with contributing to big Rust projects, or working with big Rust projects in general.

Comment (MartinFillon) [reply src/output/icons.rs]: > Sorry for being so incompetent, I have little experience with contributing to big Rust projects, or working with big Rust projects in general.

Its okay I might have said it too rudely too

**Comment by MartinFillon** [comment]
well its working now but as you can see there is still unusefull changes

**Review by MartinFillon — CHANGES_REQUESTED** [review]
see my comment please really sorry for you

**Comment by Orest58008** [comment]
Basically all I did was copy and paste spaces from the main repo and then when you pointed out I just moved them around in Emacs with vx and p. Maybe it is better to just start from scratch

**Comment by MartinFillon** [comment]
yeah really sorry

**Comment by Orest58008** [comment]
No need to be sorry. It is all my mistake for not checking properly

**Comment by Orest58008** [comment]
This time everything should be okay

**Comment by MartinFillon** [comment]
> This time everything should be okay

Dont wanna scare you, if you want in 20 min I can push it myself with you

**Comment by Orest58008** [comment]
Thanks a lot for your guidance

**Review by MartinFillon — APPROVED** [review]

**Comment by Orest58008** [comment]
Once again, thanks a lot

**Pull request merged**